frontend/public/components/utils/_alerts.scss

```diff
@@ -0,0 +1,4 @@
+.co-alert-content {
+    padding-left: 5px;
+    padding-right: 30px;
```

Comment:
nit: 2 space indentation

Reply:
Still surprised we don't have a formatter for this stuff :)